### PR TITLE
[CP-1421] Fix backup, restore and device info tests

### PR DESCRIPTION
**Pull request by Lefucjusz**
Fix of the aforementioned tests after changes
implemented in PureOS.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: test/pytest/service-desktop/test_user_data_backup_restore.py [hunk 1]
```diff
@@ -145,22 +146,25 @@ def test_backup_and_restore(harness):
     log.debug("Creating backup")
     assert run_backup(harness), "Failed to initialize backup!"
 
-    harness.connection.watch_port_reboot(20)
-    harness = get_harness_automatic(20)
+    harness.connection.watch_port_reboot(40)
+    harness = get_harness_automatic(40)
```
Comment (SP2FET) [inline test/pytest/service-desktop/test_user_data_backup_restore.py:150]: name those magic numbers :P

File: test/pytest/service-desktop/test_user_data_backup_restore.py [hunk 2]
```diff
@@ -145,22 +146,25 @@ def test_backup_and_restore(harness):
...
+    time.sleep(5)  # Without it harness fails to read from port
```
Comment (SP2FET) [inline test/pytest/service-desktop/test_user_data_backup_restore.py:151]: weird

File: test/pytest/service-desktop/test_user_data_backup_restore.py (hunk 1 above)
Comment (Lefucjusz) [reply test/pytest/service-desktop/test_user_data_backup_restore.py]: Fixed ;)

File: test/pytest/service-desktop/test_user_data_backup_restore.py (hunk 2 above)
Comment (Lefucjusz) [reply test/pytest/service-desktop/test_user_data_backup_restore.py:152]: I know, but have no idea why this happens. Such "fix" is implemented in other places in the code too.

**Review by SP2FET — APPROVED** [review]

**Review by patrycja-paczkowska — APPROVED** [review]

**Pull request merged**